chcemvediet/static/main/css/custom.sass

```diff
@@ -80,6 +80,43 @@ a:hover, a:focus
       padding-top: chv_baseline_fix($chv-fontsize-base, 1)
       overflow: hidden
 
+//
+// Casestudy link component
+//
+
+.chv-casestudy.chv-casestudy
+  display:flex
+  flex-direction:row
+  flex-wrap:wrap
+  padding: ($chv-baseline - 1px)
+  margin-bottom: $chv-baseline
+  border: solid 1px $chv-color-gray-ll
+  border-radius: 4px
+  color: #4d4d4d
+  &:hover, &:focus
+    color: darken(#4d4d4d, 20%)
+  @include clearfix()
+  > .chv-casestudy-img
+      display: block
+      float: left
+      height: ($chv-baseline * 8)
+      width: ($chv-baseline * 8)
+      margin-right: $chv-baseline
+  > .chv-casestudy-wrap
+    flex-basis: 50%
+    flex-grow:1
+    > .chv-casestudy-date
+      display: block
+      @include chv-baseline($chv-fontsize-base * 0.9, 1, 0, 0.1)
```

Comment:
@martinmacko47 teraz ked to je cele v div, tak sa mohol pouzit `chv-baseline`. Ak je text vedla obrazka tak je datum o trocha nizzsie ako obrazok (skoro neviditelny rozdiel), ten padding je tam dolezity vtedy, ked sa datum zalomi pod obrazok

Reply:
Akurat pozor, ze ak do `chv-baseline` dat padding 0.1, tak nebude sediet riadkovanie.

Ak debug mode pridas do url `?grid`, tak sa ti zobrazi mriezka s baselinami. Text by mal vzdy sediet na hlavnej baseline. Pripadne na vedlajsej baseline. Nikdy by vsak nemal sediet "v lufte" medzi nimi. Obcas to sice mame zle, ale ak sa da, mali by sme riadkovanie dodrziavat.

V tvojom pripade mas teraz text mimo baseline:

![wrong](https://user-images.githubusercontent.com/5547275/116879918-9940ef80-ac21-11eb-9291-ffe2fbefc89d.png)

Spravne by to malo byt takto:

![correct](https://user-images.githubusercontent.com/5547275/116879937-9e9e3a00-ac21-11eb-9eb2-b1614fc27ef1.png)

To dosiahnes tak, ze do `chv-baseline` das padding 0.5. Ak niekde potrebujes dat padding iny, nez nasobok 0.5, potom to musis kompenzovat opacnym paddingom niekde inde. Aby v sucte to bolo vzdy nasobok 0.5. Potom riadkovanie sedi.

Reply:
V tom pripade som pouzil `padding-top`, Priestor pod poslednym odstavcom je 1 baseline.
![Screenshot_2021-05-04 Chcemvediet sk(1)](https://user-images.githubusercontent.com/29918752/116995679-a837a880-acda-11eb-8532-35eaf8e617a1.png)